tests/components_to_test/resnet.py

```diff
@@ -22,9 +22,15 @@ def get_cifar10_dataloader(train):
 
 @non_distributed_component_funcs.register(name='resnet18')
 def get_resnet_training_components():
-    model = resnet18(num_classes=10)
+
+    def model_builder(checkpoint=False):
+        return resnet18(num_classes=10)
```

Comment:
Is `checkpoint` here only for api consistency?

Reply:
Yes...

Reply:
ok.